spec/support/cuprite_setup.rb

```diff
@@ -108,7 +108,10 @@ def register_better_cuprite(language, name: :"better_cuprite_#{language}")
 RSpec.configure do |config|
   config.around(:each, :js, type: :feature) do |example|
     # Skip if driver is explicitly requested
-    next if example.metadata[:driver]
+    if example.metadata[:driver]
+      example.run
+      next
+    end
```

Comment:
So, since before we did not call `example.run` in the around filter we did never run the test and now we are running it, but should we not set the `Capybara.javascript_driver` as well? Or is that automatically done by the `driver:` metadata?

Reply:
<img width="862" alt="Bildschirmfoto 2025-03-20 um 11 44 58" src="https://github.com/user-attachments/assets/086e6b9e-012c-4bd1-8f7b-157131f63844" />

Setting the driver is part of capybara itself, so yes ... This does what it says on the tin. Good catch!

Reply:
It's automatically done by the `driver:` metadata. It's handled by Capybara.